src/features/map/AMForceGraph.tsx

```diff
@@ -41,9 +42,9 @@ export const AMForceGraph = ({
   stitchesTheme: StitchesTheme;
 }) => {
   const fgRef = useRef<any>(null);
```

Comment:
is fgRef still needed after removing
`if (!fgRef.current) {
        return;
      }` ?

Reply:
ah, good point, it's not in use anymore i see. i'll add a comment about it in case we want to use it in the future